src/components/HomePageIconList/TopicsList.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
```

Comment:
If there are nay modifications to any of these components, the changes should be moved to data-catalog-components.

Reply:
These changes were pushed w/ this PR: https://github.com/interra/data-catalog-components/pull/5/files

I think of this as a working branch that would need to remove these components before being acceptable to be pulled in.